test-plone-5.1.x.cfg

```diff
@@ -1,9 +1,15 @@
 [buildout]
 extends =
-    https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-4.2.x.cfg
+    https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-5.1.x.cfg
     sources.cfg
 
 package-name = ftw.footer
 
-[versions]
-ftw.builder = 1.4.0
+extensions =
+    mr.developer
+
+auto-checkout =
+    ftw.testbrowser
+
+[branches]
+ftw.testbrowser = plone-5.1.x
```

Comment:
🚧 `ftw.testbrowser` has not Plone 5.1.x release yet, see https://github.com/4teamwork/ftw.testbrowser/pull/47